templates/cicd/manifest.yml

```diff
@@ -0,0 +1,23 @@
+# This manifest defines the relationship and deployment ordering of your environments.
+
+# The version of the schema
+version: {{.Version}}
+
+# This section defines the source artifacts.
+source:
+  # The name of the provider that is used to store the source artifacts.
+  provider: {{.Provider}}
+  # Additional properties that further specifies the exact location
+  # the artifacts should be sourced from.
+  properties:
+    # Repository in the format of <user_name>/<repository_name>.
+    repository: {{.Repository}}
+
+{{$length := len .Environments}}{{if gt $length 0}}
+# The deployment section defines the order the pipeline will deploy
+# to your environments
+stages: {{range $env := .Environments}}
+    - 
+      # The name of the environment to deploy to.
+      env: {{$env.Name}}
```

Comment:
We should include the applications here as well - I think. Even if we don't support anything complicated right now, we should include the name of the applications.

Reply:
Would there ever be a case where a customer deploys only a set of their applications to certain environments? Like `(app1, app2)` to `prodIAD` while `(app1, app3)` to `prodPDX`?

I wrote this schema with the assumption that all apps in a workspace deploy to all environments and I would like to keep the it simpler unless the assumption is incorrect.

Reply:
Well there could be a case where you deploy one before the other, I think.

But I see your point about having to keep everything in sync.

This is probably fine for now, to get us started. But we'll probably want to add more flexibility in the future.

Reply:
Will think about it :)